notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        budget_milli_node_hours=20000,
</pre><p>Reduce budget_milli_node_hours to minimum allowed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>